region/brinstar/red/Red Tower.json

```diff
@@ -857,7 +857,24 @@
       "link": [6, 1],
       "name": "Base",
       "notable": false,
-      "requires": []
+      "requires": [
+        {"or": [
+          "canWalljump",
+          "HiJump",
+          "canSpringBallJumpMidAir",
+          "h_canFly"
+        ]}
+      ]
+    },
+    {
+      "link": [6, 1],
+      "name": "Frozen Beetom Lure",
+      "notable": false,
+      "requires": [
+        "canTrickyUseFrozenEnemies",
+        "canCrouchJump"
```

Comment:
```suggestion
        "h_canCrouchJumpDownGrab"
```

Reply:
The room segment just below the shot blocks uses a crouchJumpDownGrab to climb up.